src/components/AppComponents/Navbar.vue

```diff
@@ -2,6 +2,11 @@
   <nav>
     <v-toolbar height="100">
       <img :src="require('@/assets/images/logo.png')" />
+      <v-progress-circular v-if="displayProgressBar"
+                           indeterminate
+                           color="primary"
+                           class="circle-progress"
+                           :size="40" />
```

Comment:
מה זה אומר? למה עשית את זה עם קבוע שהוא מקושר?

Reply:
It is a vuex state that says if the spinner should be displayed or not

Reply:
What represents the "40" number?

Reply:
the size of the spinner